Add instructions for installing and upgrading mixxx (#114)
mixxxdj/manual: source/chapters/getting_started.rst

Getting Started
***************

Installing Mixxx
================

> Once you've `downloaded Mixxx <https://mixxx.org/download/>`_, you need
> to install it on your computer.

Reviewer: Both the "Installing Mixxx" and the "Opening Mixxx" sections start with the same sentence structure.

Maybe we can add a little variation, e.g.:
```suggestion
After `downloading Mixxx from our Website <https://mixxx.org/download/>`_, you need
to install it on your computer.
```
